src/containers/NonBitGoRecoveryCoin/NonBitGoRecoveryCoin.tsx

```diff
@@ -337,6 +336,10 @@ function Form() {
     case 'tarbeth':
     case 'opeth':
     case 'topeth':
+    case 'polygon':
+    case 'tpolygon':
+    case 'bsc':
+    case 'tbsc':
```

Comment:
For bsc and tbsc, don't support eip1559 on the chain right?, Have you made changes to accomodate that?

Reply:
bsc now supports eip1559 so I used the common recover function. We can add eip1559 support for bsc in wallet platform too as a separate task